src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -120,22 +122,39 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
       refreshSubscriptionTab()
       loadNewChatId()
     }
+    ApplicationManager.getApplication()
+        .messageBus
+        .connect()
+        .subscribe(
+            LafManagerListener.TOPIC,
+            LafManagerListener {
+              ApplicationManager.getApplication().executeOnPooledThread { refreshSubscriptionTab() }
+            })
```

Comment:
is there any reason we subscribe from CodyToolWindowContent? can we move it directly to the SubscriptionTabPanel? Let's try to remove code from this class instead of adding it 😅